tests/models/test_param.py

```diff
@@ -167,18 +167,28 @@ def test_params_dict(self):
         assert pd3.dump() == {'key': 'value'}
 
         # Validate the ParamsDict
-        pd.validate()
+        plain_dict = pd.validate()
+        assert type(plain_dict) == dict
         pd2.validate()
         pd3.validate()
 
         # Update the ParamsDict
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=r'Invalid input for param key: 1 is not'):
```

Comment:
I assume this is not the complete error message? (If it is, we should change it…) It’s probably a good idea to add an extra assert to make sure the full error message is as expected.

Reply:
The rest of the error message depends on the format of the exception thrown by the JSONSchema library we use so I didn't want to be too specific here.

Reply:
Makes sense (especially since I _just_ fixed a PR yesterday because MarkupSafe changed its exception message format).